packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +541,99 @@ void main() {
+          final ScrollConfiguration configuration = ScrollConfiguration(
+           behavior: const ScrollBehavior(),
+           child: ReorderableListView(
+            scrollDirection: Axis.horizontal,
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+          ));
+
+          ScrollPhysics physics;
+          await tester.pumpWidget(
+            Builder(
+              builder: (BuildContext context) {
+                physics = configuration.behavior.getScrollPhysics(context);
+                return Container();
+              },
+            ),
+          );
+          expect(physics, isInstanceOf<BouncingScrollPhysics>());
+        }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }));
+
+        testWidgets('Default scroll physics for android/fuchsia/linux/windows', (WidgetTester tester) async {
+          final ScrollConfiguration configuration = ScrollConfiguration(
+           behavior: const ScrollBehavior(),
+           child: ReorderableListView(
+            scrollDirection: Axis.horizontal,
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+          ));
+
+          ScrollPhysics physics;
+          await tester.pumpWidget(
+            Builder(
+              builder: (BuildContext context) {
+                physics = configuration.behavior.getScrollPhysics(context);
+                return Container();
+              },
+            ),
+          );
+          expect(physics, isInstanceOf<ClampingScrollPhysics>());
+        }, variant: const TargetPlatformVariant(<TargetPlatform>{
+            TargetPlatform.android,
+            TargetPlatform.fuchsia,
+            TargetPlatform.linux,
+            TargetPlatform.windows,
+          }),
+        );
+
+        testWidgets('physics:AlwaysScrollableScrollPhysics overrides default behavior', (WidgetTester tester) async {
+          bool scrolled = false;
+          final ReorderableListView listViewDefault = ReorderableListView(
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+          );
+
+          final ReorderableListView listViewOverride = ReorderableListView(
+            physics: const AlwaysScrollableScrollPhysics(),
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+          );
+
+          // Physics is not set to always scroll - onNotification should not be called
+          await tester.pumpWidget(
+            MaterialApp(
+              home: Directionality(
+              textDirection: TextDirection.ltr,
+              child: NotificationListener<OverscrollNotification>(
+                onNotification: (OverscrollNotification message) { scrolled = true; return false; },
+                child: listViewDefault,
+              ),
+            ),
+            )
+          );
```

Comment:
nit: indentations are a little off, as well as trailing commas
```suggestion
          await tester.pumpWidget(
            MaterialApp(
              home: Directionality(
                textDirection: TextDirection.ltr,
                child: NotificationListener<OverscrollNotification>(
                  onNotification: (OverscrollNotification message) { scrolled = true; return false; },
                  child: listViewDefault,
                ),
              ),
            ),
          );
```